org.lflang/src/org/lflang/generator/rust/RustGenerator.kt

```diff
@@ -135,7 +135,7 @@ class RustGenerator(
 
         if (cargoReturnCode == 0) {
             // We still have to copy the compiled binary to the destination folder.
-            val isWindows = System.getProperty("os.name").toLowerCase().contains("win")
+            val isWindows = System.getProperty("os.name").lowercase().contains("win")
```

Comment:
@cmnrd I tested the Kotlin support by using the `lowercase` function as in #1311. It compiles with maven and works in the Epoch product. I hope this is sufficient.